hack/make/build-rpm

```diff
@@ -96,8 +96,7 @@ set -e
 			# Install runc and containerd
 			RUN ./hack/dockerfile/install-binaries.sh runc-dynamic containerd-dynamic
 		EOF
-
-		if [ "$DOCKER_EXPERIMENTAL" ]; then
+		if [[ "$VERSION" == *-dev ]] || [ -n "$(git status --porcelain)" ]; then
 			echo 'ENV DOCKER_EXPERIMENTAL 1' >> "$DEST/$version/Dockerfile.build"
 		fi
```

Comment:
Should I remove [`--define '_experimental ${DOCKER_EXPERIMENTAL:-0}'`](https://github.com/docker/docker/blob/2c620d0aa24c5f774a9115449a86b158b005bba8/hack/make/build-rpm#L119) too ?